src/components/App.tsx

```diff
@@ -598,6 +600,8 @@ class App extends React.Component<AppProps, AppState> {
                       id={this.id}
                       onImageAction={this.onImageAction}
                       renderWelcomeScreen={
+                        !this.state.isLoading &&
+                        this.props.UIOptions.welcomeScreen !== false &&
```

Comment:
do we need to explicitly check for `false` ?

Reply:
right now yes. We'd need to normalize the default value as we do with toggleTheme.

Reply:
can we not set default value for `welcomScreen` as true always if not passed so we don't need to check for `false` explicitly like we do for `canvasActions` ?